static/app/views/alerts/wizard/panelContent.tsx

```diff
@@ -147,4 +147,64 @@ export const AlertWizardPanelContent: Record<AlertType, PanelContent> = {
     ],
     illustration: diagramCrashFreeUsers,
   },
+  app_start_cold: {
+    description: t('App cold start metric track when your app takes too long to launch.'),
+    examples: [
+      t(
+        'On iOS, Apple recommends your app take at most 400ms to render the first frame.'
+      ),
+      t(
+        'On Android, the Google Play console warns you when a cold start takes longer than five seconds or a warm start longer than two seconds.'
+      ),
+    ],
+    illustration: diagramCustom,
+  },
+  app_start_warm: {
+    description: t(
+      'App warm start metric track when your app takes does not too long to launch.'
```

Comment:
```suggestion
      'App warm start metrics track how long your mobile application takes to launch.'
```

Reply:
@marandaneto, your suggestion for cold and warm starts are the same, maybe for warm we could add some extra information, like  'App warm start metrics track how long your mobile application takes to launch when already in memory.'

Reply:
@brustolin warm start changes depending on the OS, so I'd not say that since it's gonna be wrong for one of the OS, I guess making it more generic makes more sense, and the examples below are more specific.